server/src/main/scala/io/delta/sharing/server/config/ServerConfig.scala

```diff
@@ -215,7 +215,8 @@ case class SchemaConfig(
 case class TableConfig(
     @BeanProperty var name: String,
     @BeanProperty var location: String,
-    @BeanProperty var cdfEnabled: Boolean = false) extends ConfigItem {
+    @BeanProperty var cdfEnabled: Boolean = false,
+    @BeanProperty var startVersion: Long = 0) extends ConfigItem {
```

Comment:
It's probably better to define it as `Option[Long]` here so this data structure is more self-explanatory.

Reply:
NVM. Thinking twice making it optional may suggest that the start version will not be enforced.

Reply:
sg, thanks!